app/src/main/java/com/github/se/orator/ui/overview/SpeakingJobInterviewModule.kt

```diff
@@ -27,46 +23,73 @@ fun SpeakingJobInterviewModule(
+              question = "What is your target job position?",
+              placeholder = "e.g., Senior Executive",
+              testTag = "targetPositionInput"),
           InputFieldData(
               value = companyName,
-              onValueChange = { newValue: String ->
-                companyName = newValue
-              }, // Explicitly specify String
-              label = "Which company are you applying to?",
-              placeholder = "e.g McKinsey",
-              testTag = "jobInput",
-              height = 200),
+              onValueChange = { companyName = it },
+              question = "Which company are you applying to?",
+              placeholder = "e.g., McKinsey",
+              testTag = "companyNameInput"),
           InputFieldData(
-              value = skills,
-              onValueChange = { newValue: String ->
-                skills = newValue
-              }, // Explicitly specify String
-              label = "What skills or qualifications do you want to highlight?",
-              placeholder = "e.g Problem solving",
-              height = 85,
-              testTag = "skillsInput"),
+              value = interviewType,
+              onValueChange = { interviewType = it },
+              question = "What type of interview are you preparing for?",
+              placeholder = "Select interview type",
+              testTag = "interviewTypeInput",
+              isDropdown = true,
+              dropdownItems =
+                  listOf(
+                      "Phone Interview",
+                      "Video Interview",
+                      "In-Person Interview",
+                      "Panel Interview",
+                      "Group Interview",
+                      "Assessment Center")),
           InputFieldData(
-              value = feedbackType,
-              onValueChange = { newValue: String ->
-                feedbackType = newValue
-              }, // Explicitly specify String
-              label = "Do you want feedback on persuasive language, volume, or delivery?",
-              placeholder = "e.g Persuasive language",
-              height = 85,
-              testTag = "experienceInput"))
+              value = experienceLevel,
+              onValueChange = { experienceLevel = it },
+              question = "What is your experience level in this field?",
+              placeholder = "Select experience level",
+              testTag = "experienceLevelInput",
+              isDropdown = true,
+              dropdownItems =
+                  listOf("Entry-Level", "Mid-Level", "Senior-Level", "Executive-Level")),
+          InputFieldData(
+              value = jobDescription,
+              onValueChange = { jobDescription = it },
+              question = "Please provide the job description:",
+              placeholder = "Paste the job description here",
+              testTag = "jobDescriptionInput",
+              isScrollable = true,
+              height = 200.dp),
```

Comment:
Same comment for the hardcoded height as before.

Reply:
Thank you for noticing that! Solved.